client-apis/encrypt.md

```diff
@@ -60,6 +64,19 @@ and MUST be ignored.
 ### Plaintext
 
 The plaintext to encrypt.
+This MUST be a sequence of bytes.
+
+This input MAY be streamed to this operation.
+This means that bytes are made available to this operation sequentially and over time,
+and that an end to the input is eventually indicated.
+This means that:
+
+- There MUST be a mechanism for input bytes to become available to be processed.
+- There MUST be a mechanism to indicate that there are no more input bytes,
+  or whether more bytes MAY be made available in the future.
```

Comment:
I think it's worth lifting this definition into its own client-apis/streaming.md feature, or perhaps adding to structures.md. You can focus there on terminology definitions (things like the "EOF" that is a separate signal and NOT an EOF marker, the idea of "releasing" data, etc), describe more explicitly what it means for an operation to take a stream as input vs. produce a stream as output, and so on. It will also help shorten and focus the content here.

Reply:
Moved these concerns to client-api/streaming.md